frontend/src/components/SpouseForm.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { FormProvider, useForm, Controller } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { useQuery } from 'react-query';
+import { useMutation, useQueryClient } from 'react-query';
+import FormGroup from '@mui/material/FormGroup';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import InputLabel from '@mui/material/InputLabel';
+import ConsentPersonal from './ConsentPersonal';
+import ConsentTax from './ConsentTax';
+import FileDropArea from './upload/FileDropArea';
+import useAxios from '../utils/axiosHook';
+import Box from '@mui/material/Box';
+
+export const defaultValues = {
+  application: '',
+  sin: '',
+  first_name: '',
+  last_name: '',
+  middle_names: '',
+  email: '',
+  address: '',
+  city: '',
+  postal_code: '',
+  date_of_birth: '',
+  documents: [],
+  consent_personal: false,
+  consent_tax: false
+};
+
+const SpouseForm = ({ id }) => {
+  const queryClient = useQueryClient();
+  const methods = useForm({
+    defaultValues
+  });
+  const applicationId = id;
+  const { control, handleSubmit, register, watch } = methods;
+  const axiosInstance = useAxios();
+
+  const queryFn = () =>
+    axiosInstance.current
+      .get(`/api/spouse-application/${id}/initiate`)
+      .then((response) => response.data);
+
+  const { data, isLoading, isError, error } = useQuery(
+    ['application', id],
```

Comment:
this is a different object and should use a different cache key, not reusing `application`